src/types/code.ts

```diff
@@ -0,0 +1,8 @@
+import { type } from 'os';
+
+export type code = {
+  system: 'SNOMED-CT' | 'RxNorm' | 'LOINC' | 'NUBC' | 'DICOM-DCM' | 'DICOM-SOP';
+  code: string;
+  display: string;
+  value_set: string;
```

Comment:
```suggestion
  value_set?: string;
```

I think value_set should be optional